Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
the second request to server point is only needed for dev env, in prod it will be done when you dump assets afaik. however if it does not work that is something else.